submissions/DmytryjK/task_js_dom/js/script.js

```diff
@@ -0,0 +1,186 @@
+        ]
+    },
+    {
+        title : "США",
+        subtitle : "Національний заповідник Гіффорд Пінчот, Вашингтон",
+        imgSrc : "images/USA_gifford.jpg",
+        imgAlt : "USA gifford nature reserve",
+        paragraphsArray : [
+            "Національний Вулканічний Пам'ятник Гіффорд Пінчот (Gifford Pinchot) займає величезну площу – 5537,3 квадратних кілометрів у штаті Вашингтон, США. На території природоохоронного об'єкта знаходяться гори та ліси, озера та річки, але головним об'єктом все ж таки залишається вулкан Сент-Хеленс. Саме Вулкан Святої Олени дав Національному парку таку незвичайну назву.",
+            "На околицях вулкана Сент-Хеленс розташована Лава поляна – величезна територія, що зберегла наслідки багатовікових вивержень. Багатосотметрові лавові труби тут є сусідами з глибокими звивистими печерами. Найвідоміша – Мавпа печера, куди туристи допускаються лише у спеціальному спелеологічному спорядженні. Мавпа печера – має найбільшу протяжність серед лавових печер США – майже 400 метрів."
+        ]
+    },
+    {
+        title : "Канада",
+        subtitle : "Озеро Морейн",
+        imgSrc : "images/Moraine Lake Canada.jpg",
+        imgAlt : "Moraine Lake Canada",
+        paragraphsArray : [
+            "Озеро Морейн підживлюється льодовиком і досягає свого повного наповнення лише у другій половині червня. Головною принадою озера Морейн є синій колір води. Коли воно наповнене, в ньому відбиваються різні відтінки синього кольору через заломлення світла на кам'янистому дні озера.",
+            "Поверхня озера Морейн відображає, як в дзеркалі, десять величезних канадських піків, і це робить його одним з найкрасивіших місць в Канаді. Це одне з найбільш часто фотографованих місць в Канаді, що дуже славиться красою мальовничих місць і своїми пішохідними екскурсіями."
+        ]
+    },
+    {
+        title : "Франція",
+        subtitle : "Етрета, Нормандія",
+        imgSrc : "images/etretat normandy france.jpg",
+        imgAlt : "etretat normandy france",
+        paragraphsArray : [
+            "Étretat (Етрета) - невеликий курорт з прекрасними гальковими пляжами на узбережжі Алебастрового Ла Манша, одна з головних визначних пам'яток Нормандії, на півночі Франції. Він розташований біля самого пляжу, де води Англійського каналу сформували дивовижну красу прямовисні скелі з природними арками — один із найкрасивіших краєвидів Франції.",
+            ""
+        ]
+    },
+    {
+        title : "Гренландія",
+        subtitle : "Льодяний каньйон",
+        imgSrc : "images/Grenland canyon.jpg",
+        imgAlt : "Grenland canyon",
```

Comment:
```suggestion
        imgAlt : "Greenland canyon",
```

Reply:
You can add spellchecker extension to your editor ;)

Reply:
Fixed, and the Grammarly application was installed on my PC)